README.md

```diff
@@ -27,26 +27,17 @@ mvn package -DskipTests
 
 `mvn package` will assemble all the required dependencies and package into an uber jar.
 
-Pre-create Atlas models (Recommended)
-=====================================
+Create Atlas models
+===================
+NOTE: below steps are only necessary prior to Apache Atlas 2.1.0. Apache Atlas 2.1.0 will include the models. 
 
-You may need to create Atlas models to ensure all necessary models are created before running SAC and pushing metadata changes. Spark Atlas Connector provides the way to pre-create Atlas models, and it is only needed for the first time.
+You may need to create Atlas models for Spark to ensure all necessary models are created before running SAC and pushing metadata changes.
 
-Suppose Spark is installed in `<SPARK_HOME>` directory and `atlas-application.properties` is placed on `<SPARK_HOME>/conf` directory:
+The official and recommended approach for registering models in Apache Atlas is having json model file. We've co-worked with Atlas team to define official Spark models for Apache Atlas which is compatible with SAC.
```

Comment:
And delete this line. The statement after this is sufficient to explain what users need to do and why.

Reply:
Hmm... yeah I agree it's kind of verbose for end users. Let's remove this.